client/src/components/ManageAccount/ManageAccount.scss

```diff
@@ -1,50 +1,22 @@
 /*
   Wrappers
 */
-.manage-account-wrapper {
-  display: flex;
-  padding: 0;
-  .container-fluid {
-    padding: 20px 40px 0 25px !important; /* overwrite Bootstrap */ /* NOTE: The `!important` may not be necessary */
-    display: flex;
-    flex-direction: column;
-    margin: 0;
-  }
-}
 .manage-account-content {
-  a {
-    color: #9d85ef;
-  }
+  /* Hide the scrolling caused by overfow likely from bootstrap container/row/column margin/padding games */
+  overflow-x: hidden;
 }
 .user-profile {
-  flex-grow: 1;
-
   @media (min-width: 992px) {
     /* Hide Border when tables stack on one another */
     .user-profile-main {
       border-right: 1px solid rgba(112, 112, 112, 0.25);
     }
   }
-  .btn-link {
-    color: #9d85ef;
-  }
```

Comment:
**Why?**

- This style has been moved to `index.css`, because [other components rely on it](https://github.com/TACC/Frontera-Portal/pull/236#issuecomment-721198861).